<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_engine</class>
 <widget class="QWidget" name="search_engine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="search_pattern">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboCategory"/>
     </item>
     <item>
      <widget class="QPushButton" name="search_button">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="status_lbl">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="search_status">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Stopped</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget"/>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="download_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Download</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="goToDescBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Go to description page</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>601</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="enginesButton">
         <property name="text">
          <string>Search engines...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>search_pattern</sender>
   <signal>returnPressed()</signal>
   <receiver>search_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>685</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
